### PR TITLE
Added support for newer Django Filer installs

### PR DESCRIPTION
**Pull request by mkoistinen**
This fixes an issue with django-filer > 1.2.0

Please review: (@yakky or @czpython) **and** @FinalAngel  =)

### CODE REVIEW

**Comment by yakky** [comment]
:+1:

File: aldryn_newsblog/boilerplates/bootstrap3/templates/aldryn_newsblog/includes/article.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
             {% if not detail_view %}
                 <a href="{{ article.get_absolute_url }}">
             {% endif %}
-            <img src="{% thumbnail article.featured_image.image 800x450 crop subject_location=article.featured_image.subject_location %}" alt="{{ article.featured_image.alt }}" class="img-responsive">
+            <img src="{% thumbnail article.featured_image 800x450 crop subject_location=article.featured_image.subject_location %}" alt="{{ article.featured_image.alt }}" class="img-responsive">
```
Comment (FinalAngel) [inline aldryn_newsblog/boilerplates/bootstrap3/templates/aldryn_newsblog/includes/article.html]: would this fail on older projects?

Comment (mkoistinen) [inline aldryn_newsblog/boilerplates/bootstrap3/templates/aldryn_newsblog/includes/article.html]: Shouldn't.

**Comment by mkoistinen** [comment]
Rebased.

File: aldryn_newsblog/boilerplates/bootstrap3/templates/aldryn_newsblog/includes/article.html (hunk 1 above)
Comment (FinalAngel) [inline aldryn_newsblog/boilerplates/bootstrap3/templates/aldryn_newsblog/includes/article.html]: it does fail @mkoistinen this causes now problems on Aldryn sites

**Pull request merged**